cpp/arcticdb/version/test/test_sorting_info_state_machine.cpp

```diff
@@ -0,0 +1,95 @@
+/* Copyright 2023 Man Group Operations Limited
+ *
+ * Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
+ *
+ * As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+ */
+
+#include <gtest/gtest.h>
+
+#include <bitmagic/bm.h>
+#include <bitmagic/bmserial.h>
+#include <arcticdb/util/buffer.hpp>
+#include <arcticdb/util/bitset.hpp>
+
+#include <msgpack.hpp>
+#include <arcticdb/util/test/generators.hpp>
+#include <arcticdb/version/version_store_api.hpp>
+
+
+TEST(Sorting, ExistingUnknown) {
+    using namespace arcticdb;
+    using namespace arcticdb::proto;
```

Comment:
You can put these two outside the test body (at the top of the file) so they don't need to be repeated.

Similarly, consider `using SortedValue = proto::descriptors::SortedValue` at the top of the file, instead of `descriptors::` everywhere.

I would go further by declaring `constexpr UNKNOWN = proto::descriptors::SortedValue::UNKNOWN;` at the top for all four enum items, so they don't need to be qualified in this file.

Reply:
I agree with all. Pushed all the changes